src/System.Text.Json/src/System.Text.Json.csproj

```diff
@@ -70,14 +70,15 @@
     <Compile Include="System\Text\Json\Writer\Utf8JsonWriter.WriteValues.String.cs" />
     <Compile Include="System\Text\Json\Writer\Utf8JsonWriter.WriteValues.UnsignedNumber.cs" />
   </ItemGroup>
-  <ItemGroup Condition="'$(TargetGroup)' != 'netstandard'">
-    <ProjectReference Include="..\..\System.Memory\src\System.Memory.csproj" />
-    <ProjectReference Include="..\..\System.Runtime.Extensions\src\System.Runtime.Extensions.csproj" />
+  <ItemGroup Condition="'$(TargetsNETStandard)' != 'true'">
+    <Reference Include="System.Diagnostics.Debug" />
+    <Reference Include="System.Resources.ResourceManager" />
+    <Reference Include="System.Runtime" />
+    <Reference Include="System.Runtime.Extensions" />
+    <Reference Include="System.Text.Encoding.Extensions" />
+    <Reference Include="System.Threading.Tasks" />
```

Comment:
Can you please explain this change and the need for these new references?

Reply:
I removed the reference to System.Private.CoreLib.  Since this is now compiling against references it needs to reference the public location rather than the implementation location.